docs/bias.css

```
@@ -0,0 +1,5 @@
+img[src$="align=right"] {
```

https://www.w3schools.com/cssref/css_selectors.asp

With the rule above you are free to extend image URLs with `?align=right` or `align=right`. A more strict version:
```suggestion
img[src$="?align=right"] {
```

You could even implement multiple "pseudo classes" with 
```suggestion
img[src$~="align=right"] {
```

With this hack the rule can move to `_assets/css/tyle.scss` to have it available on all pages. This way you currently(!) might not need page specific CSS files any more.